File: sdks/thunkmetrc-go/wrapper/models/transfer_template_delivery_transporter_detail.go
package models

type TransferTemplateDeliveryTransporterDetail struct {
    ActualDriverStartDateTime *string `json:"ActualDriverStartDateTime,omitempty"`
    DriverLayoverLeg *string `json:"DriverLayoverLeg,omitempty"`
    DriverName string `json:"DriverName,omitempty"`
    DriverOccupationalLicenseNumber string `json:"DriverOccupationalLicenseNumber,omitempty"`
    DriverVehicleLicenseNumber string `json:"DriverVehicleLicenseNumber,omitempty"`
    VehicleLicensePlateNumber string `json:"VehicleLicensePlateNumber,omitempty"`
    VehicleMake string `json:"VehicleMake,omitempty"`
    VehicleModel string `json:"VehicleModel,omitempty"`
}

File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/transfer_template_delivery_transporter_detail.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class TransferTemplateDeliveryTransporterDetail(TypedDict, total=False):
    ActualDriverStartDateTime: str
    DriverLayoverLeg: str
    DriverName: str
    DriverOccupationalLicenseNumber: str
    DriverVehicleLicenseNumber: str
    VehicleLicensePlateNumber: str
    VehicleMake: str
    VehicleModel: str
